ci-operator/step-registry/assisted/baremetal/gather/assisted-baremetal-gather-commands.sh

```diff
@@ -42,7 +42,7 @@ cd /home/assisted
 source /root/config
 
 # Get sosreport including sar data
-sos report --batch --tmp-dir /tmp/artifacts \
+sos report --batch --tmp-dir /tmp/artifacts --all-logs \
```

Comment:
consider rebasing interactively the commits in this PR so that it will not show those changes
e.g.: ``git rebase -i HEAD~3`` and choosing which message that unified commit will have